src/encoded/schemas/library.json

```diff
@@ -242,29 +259,60 @@
             "title": "Size selection method",
             "description": "A short description or reference of the size selection protocol used in library preparation, if applicable.",
             "type": "string",
-            "XXXenum": [
-                "Agencourt AMPure XP",
-                "BluePippin",
-                "gel",
-                "see document",
-                "SPRI beads"
-            ],
-            "format": "semi-controlled"
+            "oneOf": [
+                {
+                    "enum": [
+                        "agarose gel extraction",
+                        "AMPure XP bead purification",
+                        "BluePippin",
+                        "BluePippin 2-10Kb",
+                        "dual SPRI",
+                        "E-Gel",
+                        "gel",
+                        "gel followed by Amicon filters",
+                        "Invitrogen EGel EX 2% agarose (Cat# G402002)",
+                        "Pippin HT",
+                        "Sera-mag SpeedBeads",
+                        "SPRI beads"
+                    ]
+                },
+                {
+                    "comment": "Historical size selection methods, not valid for new submissions.",
+                    "permission": "import_items",
+                    "enum": [
+                        "Only RNAs greater than 200 nucleotides. The inserts for the library will vary from ~ 100 - 700 base pairs.",
+                        "Only RNAs greater than 200 nucleotides. The insert for the library will vary from ~ 100 - 700 base pairs.",
+                        "SPRI beads AMPURE",
+                        "other"
+                    ]
+                }
+            ]
         },
         "lysis_method": {
             "title": "Lysis method",
-            "description": "A short description or reference of the cell lysis protocol used in library preparation, if applicable",
+            "description": "A short description or reference of the cell lysis protocol used in library preparation, if applicable.",
             "type": "string",
-            "XXXenum": [
-                "Ambion mirVana",
-                "miRNeasy Mini kit (QIAGEN cat#:217004)",
-                "n/a",
-                "Qiagen #74204",
-                "QIAGEN DNeasy Blood & Tissue Kit",
-                "see document",
-                "Trizol (LifeTech cat#: 15596-018)"
-            ],
-            "format": "semi-controlled"
+            "oneOf": [
+                {
+                    "enum": [
+                        "ATAC buffer",
+                        "Clontech UltraLow for Illumina sequencing",
+                        "RIPA",
+                        "SDS",
+                        "[NPB (5% BSA (Sigma), 0.2% IGEPAL-CA630 (Sigma), cOmplete (Roche), 1mM DTT in PBS)]",
```

Comment:
I am wondering why not to move the long and super specimport_only" enums?
like: "[NPB (5% BSA (Sigma), 0.2% IGEPAL-CA630 (Sigma), cOmplete (Roche), 1mM DTT in PBS)]",
                        "72 degrees for 3 minutes in the presence of Triton"
"0.01% digitonin"

instead to put Digitonin, Triton ?